### DISCUSSION TITLE
Custom device update?

### DISCUSSION THREAD

**Discussion by SpuriousGer** [HASwitchPlate/HASPone#160] [General]
Hi Luna,

I watched the stream with you and Dr. Zzz a while back and you mentioned that there was some thinking around choosing a different provider for the screens. I guess the shortage of chips, etc. doesn't help, but has there been any progress?

Thank you very much!

**Comment by aderusha** [comment]
Two developments on that front:

* There is a separate project called [OpenHASP](https://github.com/HASwitchPlate/openHASP) which utilizes a range of LCDs, but which is not using blueprints etc for configuration.  It's an advanced solution under rapid development, but you'll have to dig deep as the configuration is very much manual, YAML-based, and prone to changing.  That solution (currently) has no PCB or printable enclosure, but there is support for some off-the-shelf solutions which you can buy to provide a complete enclosure.  Good solution for advanced users who want to invest the time in developing customized screens.
* Nextion has released a new line of LCDs called the "Discovery" series to deal with the parts shortage.  Support for Discovery series was added to the HASPone project last week, which brings them to feature parity with the rest of the project.  This means simple installation and configuration through blueprints and no YAML or coding is required.
